src/subsystems.jl

```diff
@@ -124,14 +131,43 @@ end
 """
 Return a Vector of subsystem names that contain the component.
 """
-function get_participating_subsystems(
+function get_assigned_subsystems(
     data::SystemData,
     component::InfrastructureSystemsComponent,
 )
     uuid = get_uuid(component)
     return [k for (k, v) in data.subsystems if uuid in v]
 end
 
+"""
+Return true if the component is assigned to any subsystems.
+"""
+function is_assigned_to_subsystem(
```

Comment:
```suggestion
function is_assigned_to_a_subsystem(
```
or 
```suggestion
function is_assigned_subsystems(
```



Otherwise it could be confused with the query to wether something is attach to a specific subsystem